packages/protocol/migrations_sol/create_and_migrate_anvil_devchain.sh

```diff
@@ -6,12 +6,15 @@ START_TIME=$SECONDS
 
 export ANVIL_PORT=8546
 
+echo "Forge version: $(forge --version)"
+
 # TODO make this configurable
 FROM_ACCOUNT_NO_ZERO="f39Fd6e51aad88F6F4ce6aB8827279cffFb92266" # This is Anvil's default account (1)
 export FROM_ACCOUNT="0x$FROM_ACCOUNT_NO_ZERO"
 
 # Create temporary directory
 TEMP_FOLDER="$PWD/.tmp"
+rm -rf $TEMP_FOLDER
```

Comment:
Nit: Functionally, this is basically equivalent, but I like the `if` syntax because it makes it easier to understand what the desired behaviour is.

```suggestion
if [ -d "$TEMP_FOLDER" ]; then
    # Remove temporary directory first it if exists
    echo "Removing existing temporary folder..."
    rm -rf $TEMP_FOLDER
fi
```

I have the same syntax in [`deploy_libraries.sh#L8C1-L12C3` ](https://github.com/celo-org/celo-monorepo/blob/3588610af2589b8daf3f917d4561ebb512dcba22/packages/protocol/migrations_sol/deploy_libraries.sh#L8C1-L12C3).